python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -130,15 +130,15 @@ class MimicExplainer(BlackBoxExplainer):
     :type model_task: str
     :param reset_index: Uses the pandas DataFrame index column as part of the features when training
         the surrogate model.
-    :type reset_index: bool
+    :type reset_index: str
```

Comment:
The doc string should provide explanation on the three options and what they do.

Reply:
oops, good catch!